rmw_opensplice_cpp/src/functions.cpp

```diff
@@ -20,27 +20,26 @@
 #include <set>
 #include <sstream>
 #include <stdexcept>
-
-#include <ccpp_dds_dcps.h>
-#include <dds_dcps.h>
-#include <u_instanceHandle.h>
+#include <string>
 
 #ifdef _WIN32
 #include <stdlib.h>  // For _putenv_s.
 #endif
 
-#include <rmw/allocators.h>
-#include <rmw/error_handling.h>
-#include <rmw/impl/cpp/macros.hpp>
-#include <rmw/rmw.h>
-#include <rosidl_generator_c/message_type_support.h>
-#include <rosidl_generator_c/service_type_support.h>
-#include <rosidl_typesupport_opensplice_cpp/identifier.hpp>
-#include <rosidl_typesupport_opensplice_cpp/impl/error_checking.hpp>
-#include <rosidl_typesupport_opensplice_cpp/message_type_support.h>
-#include <rosidl_typesupport_opensplice_cpp/service_type_support.h>
+#include <ccpp_dds_dcps.h>
+#include <dds_dcps.h>
+#include <u_instanceHandle.h>
```

Comment:
That's less than desirable. I think they might be ok in the other ones because we have some preprocessor stuff surrounding them. Maybe it just gives a pass to anything which is surrounded by preprocessor directives.